enums.h

```diff
@@ -2303,6 +2303,21 @@ typedef enum placeflag {
 	PLACE_UNIQUE  = 4,
 } placeflag;
 
+typedef enum mienemy_type {
+	TARGET_MONSTERS = 0,
+	TARGET_PLAYERS  = 1,
+	TARGET_BOTH     = 2,
+} mienemy_type;
+
+/*
+Looks like someone treated hex values as binary, so 0x10 came after 0x01, that's why we have 1 and 16, they did the same thing with affix_item_type

```

Comment:
:laughing:

Reply:
Wow! How do you tracked down such a unexpected scenario? One should have to use some very sophisticated method of modeling programmer's thoughts.

Reply:
well we never knew what "GOE" meant and it suddenly hit me ;p and it was always weird the valeus were 0/1/16 lol